config/routes.rb

```diff
@@ -1,8 +1,9 @@
 Rails.application.routes.draw do
-  root  'urls#new'
+  root  'urls#index'
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.